crates/core/src/ring.rs

```diff
@@ -46,6 +43,9 @@ use crate::{
 
 mod connection_manager;
 pub(crate) use connection_manager::ConnectionManager;
+mod location;
```

Comment:
As a recommendation for future PRs, is always better moving things in a separate PR for reviewing, since is harder to see the exact changes otherwise.